source/configuration_manual/mail_location/imapc/index.rst

```diff
@@ -0,0 +1,353 @@
+.. _imapc_settings:
+
+===================
+Imapc Configuration
+===================
+
+See :ref:`Imapc <imapc_mbox_format>` for a technical description of Dovecot's
+imapc mailbox format.
+
+Mail Location Configuration Examples
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: none
+
+  # In-memory index files:
+  mail_location = imapc:
+
+  # Store index files locally:
+  mail_location = imapc:~/imapc
+
+Connection Settings
+^^^^^^^^^^^^^^^^^^^
+
+.. _setting-imapc_cmd_timeout:
```

Comment:
the `setting` stuff should be in `settings/core.rst`

Reply:
See above: settings don't have to live on core settings page and, if not, then the settings link should go to the setting itself, not a stub entry with yet another link.